bats/sql-commit.bats

```diff
@@ -115,4 +111,56 @@ teardown() {
     run dolt sql -q "SELECT * from dolt_commits ORDER BY Date DESC;"
     [ $status -eq 0 ]
     [[ "$output" =~ "Commit1" ]] || false
+}
+
+@test "DOLT_COMMIT updates session variables and keeps dolt log fresh" {
+    run dolt sql << SQL
+SELECT DOLT_COMMIT('-a', '-m', 'Commit1');
+SELECT * FROM dolt_log;
+SQL
+
+    [ $status -eq 0 ]
+    [[ "$output" =~ "Commit1" ]] || false
+}
+
+@test "DOLT_COMMIT updates session variables and keeps dolt_diff fresh" {
+    run dolt sql << SQL
+SELECT DOLT_COMMIT('-a', '-m', 'Commit1');
+SELECT count(*) FROM dolt_diff_test WHERE to_commit = hashof('head');
+SQL
+
+    [ $status -eq 0 ]
+    # Represents that the diff table marks a change from the recent commit.
+    [[ "$output" =~ "1" ]] || false
```

Comment:
Doesn't dolt_commit() return  a hash? This could match it by accident

Reply:
Good catch.